dev/com.ibm.websphere.appserver.features/visibility/public/grpcClient-1.0/resources/l10n/io.openliberty.grpcClient-1.0.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature provides a gRPC client which can be used by applications to consume gRPC services.
```

Comment:
Change,
"This feature provides a gRPC client which can be used by applications to consume gRPC services."
to,
"This feature provides a gRPC client, which can be used by applications to use gRPC services."

Reply:
Hi @ManasiGandhi - isn't "which can be **used** by applications to **use** gRPC services" a little redundant?  Is there a better translation-friendly verb we can use?

Reply:
"This feature provides a gRPC client, which can be used by applications to call gRPC services."